README.rst

```diff
@@ -6,7 +6,7 @@ Avocado is a set of tools and libraries to help with automated testing.
 
 One can call it a test framework with benefits.  Native tests are
 written in Python and they follow the unittest
-(https://docs.python.org/2.7/library/unittest.html) pattern, but any
+(https://docs.python.org/3.7/library/unittest.html) pattern, but any
```

Comment:
Given that on travis we test Avocado up to 3.6, then maybe the link here should point to 3.6.x unittest docs instead?

Reply:
That's a good point.  We do actually advertise the Python versions we support on `setup.py` too, so you have a strong point.

Anyway, I'm basically using Python 3.7, so the lack on CI for 3.7 needs fixing (and was mostly a Travis-CI limitation a while ago).